### PR TITLE
feat(Shikimori): add presence

### PR DESCRIPTION
**Pull request by shiseni**

### CODE REVIEW

**Review by EncryptedDev — COMMENTED** [review]
Upload proof of the presence working

**Comment by alex-ench** [comment]
> Upload proof of the presence working

https://i.imgur.com/gpi5wbw.png

File: websites/S/Shikimori/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "author": {
+    "name": "SHISEN",
+    "id": "257151906038677504"
+  },
+  "service": "Shikimori",
+  "description": {
+    "en": "Online encyclopedia of anime and manga. Full list of manga in Russian. Shikimori.one is the best anime chat forum.",
```
Comment (Syjalo) [inline websites/S/Shikimori/dist/metadata.json:9]:
```suggestion
    "en": "Shikimori is online encyclopedia of anime and manga. The full list of popular manga in Russian. Shikimori.one is the best anime forum to communicate in.",
```

**Review by Syjalo — CHANGES_REQUESTED** [review]

File: websites/S/Shikimori/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "ru": "Шикимори - онлайн энциклопедия аниме и манги. Полный список популярной манги на русском. Shikimori.one - лучший аниме-форум для общения."
```
Comment (Syjalo) [inline websites/S/Shikimori/dist/metadata.json:10]:
```suggestion
    "ru": "Шикимори — онлайн энциклопедия аниме и манги. Полный список популярной манги на русском. Shikimori.one — лучший аниме-форум для общения."
```
To comply with the Russian language rules

**Review by Syjalo — CHANGES_REQUESTED** [review]

File: websites/S/Shikimori/dist/metadata.json (hunk 1 above)
Comment (ImRodry) [reply websites/S/Shikimori/dist/metadata.json:9]: First sentence has grammar incoherencies

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**